docs/tutorial/kymographs.rst

```diff
@@ -197,18 +259,23 @@ We can downsample channel data according to the lines in a kymo. We can use
 This returns a list of start and stop timestamps that can be passed directly to :func:`~lumicks.pylake.channel.Slice.downsampled_to`,
 which will then return a :class:`~lumicks.pylake.channel.Slice` with a datapoint per line::
 
-    downsampled = f.force1x.downsampled_over(line_timestamp_ranges)
+    force = file.force1x
+    downsampled = force.downsampled_over(line_timestamp_ranges)
+
+    plt.figure()
+    force.plot(label="high frequency")
+    downsampled.plot(start=force.start, label="downsampled like kymo")
+    plt.legend()
+    plt.show()
+
+.. image:: ./figures/kymographs/force_downsampled_like_kymo.png
 
 There is also a convenience function to plot a kymograph along with a downsampled force trace::
 
-    kymo.plot_with_force("1x", "green")
+    kymo.plot_with_force("1x", "green", adjustment=lk.ColorAdjustment(0, 15))
```

Comment:
Would be nice to reference this function in the text so they have something to click that will take them to the relevant API docs page.

Reply:
good point. I also really don't like the implementation of this one (I think I modeled it after FD curve construction). Maybe for a separate PR deprecating this and having instead `kymo.plot_with_channel(Slice, color)`